yetanotherco/aligned_layer
# feat (batcher): serialize different proof types

Batcher now accepts any proof type (but verifies only sp1).

# To test

1.  Start anvil, aggregator, operator & batcher (make sure to have .env file with aws credentials)
2. Run `make batcher_send_dummy_task PROVING_SYSTEM=<sp1|plonk_bls12_381|plonk_bn254|groth16_bn254>` five times (or change config.yaml to require less proofs)

Batch with different proof types can be downloaded from [here](https://storage.alignedlayer.com/99ef98ad02f8a0e14fd75026eeb500b2e17911996058fd5e917815f4c5836ebe.json)
